src/scss/colors.module.scss

```diff
@@ -66,6 +66,17 @@ $color-infobox-alert-dark: #802509;
 $color-error-light: #f9ded9;
 $color-error-dark: #e35942;
 
+
+// Six Color Palette
```

Comment:
Could you please add a prefix to indicate the usage of this palette? Maybe something like `county-` for `color-county-palette-100`?

Reply:
I think it is for use outside of this chart. I could call it `qualitative-color-palette`?

Reply:
Ah, got it! That would be great -- just something a little more descriptive 👍

Reply:
Renamed! [35eed6a](https://github.com/COVID19Tracking/website/pull/993/commits/35eed6ac70c423b877e29e080d2f1427051174a7)